justfile

```diff
@@ -66,12 +66,9 @@ new-js-lintrule rulename:
 
 # WIP: Creates a new css lint rule in the given path, with the given name. Name has to be camel case.
 new-css-lintrule rulename:
-  cargo run -p xtask_codegen -- new-css-lintrule --kind=css --name={{rulename}}
-  cargo codegen analyzer
-  cargo codegen-configuration
-  just gen-bindings
-  just format
-# TODO: lintdoc, website, cargo doc
+  cargo run -p xtask_codegen -- new-lintrule --kind=css --name={{rulename}}
+  just gen-lint
+  just documentation
```

Comment:
`just documentation` runs the `lintdoc` job -  which is run by `gen-lint` - and it checks the cargo docs. I don't think we need to run the cargo docs